policy/modules/system/userdomain.if

```diff
@@ -124,6 +124,12 @@ template(`userdom_base_user_template',`
 	# kdeinit wants systemd status
 	init_get_system_status($1_t)
 
+	ifdef(`init_systemd',`
```

Comment:
I don't think this should be in a systemd block. It has potential use outside of systemd. Otherwise looks good.

Reply:
done